.github/workflows/publish-sdk.yaml

```diff
@@ -31,4 +33,4 @@ jobs:
           echo "MAVEN_PASSWORD=${MAVEN_PASSWORD}" >> ~/.gradle/gradle.properties
 
       - name: Publish
-        run: make publish-release
+        run: make check-maven-credentials-and-publish
```

Comment:
Instead, I've switched the makefile target to one that publishes without testing, as we handle testing via the reusable workflow. ([Makefile target definition](https://github.com/Eppo-exp/android-sdk/blob/aaron/fix-publish/Makefile#L53))